spec/spec_helper.rb

```diff
@@ -5,6 +5,12 @@
 
 # Run Coverage report
 require 'solidus_dev_support/rspec/coverage'
+require 'webmock/rspec'
+require 'vcr'
```

Comment:
After a discussion IRL @abhishekgupta5 and I agreed we don't have to use VCR on this extension.

Reply:
Removed in https://github.com/nebulab/solidus_pay_tomorrow/pull/3/commits/59d028bad7d0e18d9404ccfff023ed20cc3a5f80

The specs use stub now